Description: Update logfile paths to log to /var/log/ulog/*
 The standard configuration file sends logs to a number of files under
 /var/log/. To keep things tidy, including the logrotate script, move all the
 logs into a common parent directory.
Forwarded: not-needed
Author: Chris Boot <debian@bootc.net>
Last-Update: 2013-04-27

Index: ulogd2/ulogd.conf.in
===================================================================
--- ulogd2.orig/ulogd.conf.in	2013-04-01 12:02:36.908069963 +0100
+++ ulogd2/ulogd.conf.in	2013-04-01 12:03:58.909966621 +0100
@@ -179,25 +179,24 @@
 socket_path="/tmp/nuauth_ulogd2.sock"
 
 [emu1]
-file="/var/log/ulogd_syslogemu.log"
+file="/var/log/ulog/syslogemu.log"
 sync=1
 
 [op1]
-file="/var/log/ulogd_oprint.log"
+file="/var/log/ulog/oprint.log"
 sync=1
 
 [gp1]
-file="/var/log/ulogd_gprint.log"
+file="/var/log/ulog/gprint.log"
 sync=1
 timestamp=1
 
 [xml1]
-directory="/var/log/"
+directory="/var/log/ulog/"
 sync=1
 
 [pcap1]
-#default file is /var/log/ulogd.pcap
-#file=/var/log/ulogd.pcap
+file=/var/log/ulog/ulogd.pcap
 sync=1
 
 [mysql1]
@@ -276,7 +275,7 @@
 
 [nacct1]
 sync = 1
-#file = /var/log/ulogd_nacct.log
+file = /var/log/ulog/nacct.log
 
 [mark1]
 mark = 1
